S2VX.Game/Story/Note/Approach.cs

```diff
@@ -13,7 +13,7 @@ public class Approach : CompositeDrawable, IComparable<Approach> {
         [Resolved]
         private S2VXStory Story { get; set; } = null;
 
-        private RelativeBox[] Lines { get; set; } = new RelativeBox[4]
+        public List<RelativeBox> Lines { get; } = new List<RelativeBox>
```

Comment:
FYI - I also do this (but not changing it to public in #217 ). Assuming this goes in after that change, you'll need to rebase and fix conflicts.

Reply:
Ok, that's fine. I'll merge master and take care of conflicts. I'll also probably need to move around Update logic for HoldNote because of the AlwaysPresent lag issue